pstatus-notifications-ktor/build.gradle

```diff
@@ -62,6 +62,17 @@ dependencies {
     implementation "com.expediagroup:graphql-kotlin-schema-generator:6.0.0"
     implementation "io.ktor:ktor-server-netty:2.1.0"
     implementation "io.ktor:ktor-client-content-negotiation:2.1.0"
+
+    implementation("org.jeasy:easy-rules-core:4.1.0")
```

Comment:
I think this bug fix is mixed up with the easy-rules stuff.  Do you need to include both in this PR?

Reply:
Not sure how it got in there. Will remove it.